<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_destroy">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Destroy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_host">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>2</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QDockWidget" name="netlog">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>532</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <widget class="QTextBrowser" name="txt_net_log">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>511</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Network log: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clear">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_attack">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ip_port2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_move">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_place">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Place</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_ip_port">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_join">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
